### PR TITLE
SG for app, s3 backet, cicd.service

### PR DESCRIPTION
**Pull request by achudilovski**

### CODE REVIEW

File: files/startup_script_jenkins.sh [hunk 1]
```diff
@@ -49,4 +49,11 @@ sed -i "s/subnet-id/$SUBNET_ID/g" /data/jenkins/jenkins_home/config.xml
 ## Run docker
 cd /data/docker
 sudo docker build -t cicd-jenkins -f jenkins.dockerfile .
-sudo docker-compose up -d
\ No newline at end of file
+sudo docker-compose up -d
```
Comment (publius-github) [inline files/startup_script_jenkins.sh]: We dont need to start docker compose, and start service in one time. This line should be removed

File: terraform/00-cicd-jenkins/04-security-group.tf [hunk 2]
```diff
@@ -49,3 +49,48 @@ resource "aws_security_group" "jenkins" {
...
+
+  ingress {
+    from_port   = 50504
+    to_port     = 50504
+    protocol    = "tcp"
+    cidr_blocks = "${var.ip_white_list}"
+  }
+
+  ingress {
+    from_port   = 637
```
Comment (publius-github) [inline terraform/00-cicd-jenkins/04-security-group.tf]: 6379 port

File: terraform/00-cicd/02-s3.tf [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
-#   bucket = "my-tf-test-bucket"
-#   acl    = "private"
-#   tags = {
-#     Name        = "My bucket"
-#     Environment = "Dev"
-#   }
-# }
-
+ resource "aws_s3_bucket" "simple-testing-capabilities-test-bucket" {
+   bucket = "simple-testing-capabilities-test"
```
Comment (publius-github) [inline terraform/00-cicd/02-s3.tf]: simple-testing-capabilities-tests, please
We will store tests result in this bucket

**Review by achudilovski — COMMENTED** [review]
Поправил

**Review by publius-github — APPROVED** [review]

**Pull request merged**